iosHyperskillApp/Podfile.lock

```diff
@@ -143,4 +143,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: accdcc032fac587960d2d3fbdcf802558b0caf3f
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.11.3
```

Comment:
@vladkash why downgrade?

Reply:
It was updated after `pod install --repo-update`, instead of update I should install it again, yes?

Reply:
It would be best to work with `Bundler`, not directly with `pod`. To update local Ruby gems run `bundle update`.
`COAPODS: 1.12.0` - the version that uses GitHub Actions

Reply:
@ivan-magda I reverted this change